libs/SalesforceSDKCore/SalesforceSDKCore/Classes/NativeLoginManagerInternal.swift

```diff
@@ -39,25 +39,68 @@ let maximumPasswordLengthInBytes = 16000
 /// See https://developer.apple.com/documentation/swift/importing-swift-into-objective-c#Import-Code-Within-a-Framework-Target
 @objc(SFNativeLoginManagerInternal)
 public class NativeLoginManagerInternal: NSObject, NativeLoginManager {
+        
     @objc public let clientId: String
     @objc public let redirectUri: String
     @objc public let loginUrl: String
+    
+    ///   The Google Cloud project reCAPTCHA Key's "Id" as shown in Google Cloud Console under
+    ///   "Products & Solutions", "Security" and "reCAPTCHA Enterprise"
+    private let reCaptchaSiteKeyId: String?
+    
+    ///   The Google Cloud project's "Id" as shown in Google Cloud Console
+    private let googleCloudProjectId: String?
+    
+    ///  Specifies if reCAPTCHA uses the enterprise license
+    private let isReCaptchaEnterprise: Bool
+    
     let scene: UIScene?
     
-    struct AuthorizationResponse: Codable {
+    /// A structure for the Headless Identity API's authorization endpoint response
+    private struct AuthorizationResponseBody: Codable {
         let sfdc_community_url: String
         let sfdc_community_id: String
         let code: String
     }
     
-    @objc public init(clientId: String, redirectUri: String, loginUrl: String, scene: UIScene?) {
+    @objc public init(
+        clientId: String,
+        redirectUri: String,
+        loginUrl: String,
+        scene: UIScene?
+    ) {
```

Comment:
I keep my line wrap at 80 since I primarily use my laptop's display and a few Xcode tool windows open here and there, but they invented scrollbars for a reason. 💡 I'll revert this.